# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvrec/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./divx4/divx4file_core.c.orig	2005-06-02 19:26:49.059954704 -0400
+++ ./divx4/divx4file_core.c	2005-06-02 19:28:07.248417472 -0400
@@ -275,6 +275,7 @@
 	if(divx4c->dlh)
 		dlclose(divx4c->dlh);
 */
+	return;
 }
 
 void divx4file_core_nullvideo(struct file_core_t * fct)
